test/duration_test.exs

```diff
@@ -1,10 +1,21 @@
 defmodule DurationTests do
   use ExUnit.Case, async: true
   use Timex
-  doctest Timex.Duration
+  doctest Timex.Duration, except: [{:now, 0}, {:now, 1}]
```

Comment:
This is the only way I could find to get an example in the documentation for `now/0` and `now/1` that elegantly explained the behavior of the function. A doctest that actually runs would be clumsy, and not actually serve well as documentation, in my opinion. See the new tests in the `now` describe block of `test/duration_test.exs`. I'd love to be proven wrong on this point through a more elegant doctest that isn't subject to a race condition or tautological, however!